joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/ClosureTests.scala

```diff
@@ -127,14 +127,64 @@ class ClosureTests extends PhpCode2CpgFixture {
     val cpg = code(
       """<?php
         |$captured = 5
-        |$x = fn ($value) => $value + $captured;
+        |$x = fn ($value) => $value + $captured + $captured + $captured;
         |""".stripMargin,
       fileName = "foo.php"
     )
 
     "not leave orphan identifiers" in {
       cpg.identifier.filter(node => Try(node.astParent.toList).isFailure).toList shouldBe Nil
     }
+
+    "local variable $captured exists" in {
```

Comment:
More tests needed:
1. Capturing of multiple parameters/locals into an arrow function
2. Recursive capturing of arrow function in arrow function